### PR TITLE
java 2nd assignment

### PR DESCRIPTION
**Pull request by saicharan-zemoso**
completed java 2nd assignment

### CODE REVIEW

File: java assignments/assignments/assignment2/SearchAlphabets.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class SearchAlphabets {
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner(System.in);
+        System.out.println("enter the string to search");
+        String string=new String(sc.nextLine().toLowerCase());
+        byte b[]=string.getBytes();
+        int count=26;
+        boolean findAlphas[]=new boolean[26];
+        for(int i=0;i<26;i++)
```
Comment (gunazemoso) [inline java assignments/assignments/assignment2/SearchAlphabets.java:14]: there is a method in java for this Arrays.fill please check once.

File: java assignments/assignments/assignment2/SearchAlphabets.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        if(count==0)
+        {
+            System.out.println(string+" contains all alphabets");
+        }
+        else {
+            System.out.println(string+" does not contain all alpha bets");
+        }
+
+    }
+}
```
Comment (gunazemoso) [inline java assignments/assignments/assignment2/SearchAlphabets.java:40]: i dont see any time ans space complexity here requested in assignment . please add them

File: java assignments/assignments/assignment2/SearchAlphabets.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+import java.util.Scanner;
+
+public class SearchAlphabets {
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner(System.in);
+        System.out.println("enter the string to search");
+        String string=new String(sc.nextLine().toLowerCase());
+        byte b[]=string.getBytes();
```
Comment (gunazemoso) [inline java assignments/assignments/assignment2/SearchAlphabets.java:11]: this can be done without converting into bytes also please check once.

Comment (saicharan-zemoso) [reply java assignments/assignments/assignment2/SearchAlphabets.java]: i have added the alternate method too and commented that code

File: java assignments/assignments/assignment2/SearchAlphabets.java (hunk 2 above)
Comment (saicharan-zemoso) [reply java assignments/assignments/assignment2/SearchAlphabets.java:73]: I have added the time and space complexity for the both methods now.

**Review by gunazemoso — APPROVED** [review]

**Pull request merged**